src/OpenAL/OpenToolkit.OpenAL/ALContext.cs

```diff
@@ -13,23 +14,23 @@
 
 namespace OpenToolkit.Audio.OpenAL
 {
-    public struct ALContextHandle : IEquatable<ALContextHandle>
+    public struct ALContext : IEquatable<ALContext>
     {
-        public static readonly ALContextHandle Zero = new ALContextHandle(IntPtr.Zero);
+        public static readonly ALContext Zero = new ALContext(IntPtr.Zero);
```

Comment:
Ah here its Zero, in Device its Null ;0
but
Just a question. Would `NullContext` sound better here?

Reply:
It should probably be be `Null` or `NullContext`. Probably just `Null` and to contrast that `NullDevice` because the null device is often just the default device in OpenAL

Reply:
It's `Null` now